src/ZendeskApi.Client/Models/CursorPaginatedIterator.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using ZendeskApi.Client;
+using ZendeskApi.Client.Responses;
+
+public class CursorPaginatedIterator<T> : IEnumerable<T>
```

Comment:
can we add a namespace to this class? And then move it into the Pagination folder?